ecs-cli/modules/commands/compose/compose_command.go

```diff
@@ -103,6 +103,10 @@ func createCommand(factory composeFactory.ProjectFactory) cli.Command {
 		Name:   "create",
 		Usage:  "Creates an ECS task definition from your compose file. Note that we do not recommend using plain text environment variables for sensitive information, such as credential data.",
 		Action: compose.WithProject(factory, compose.ProjectCreate, false),
+		Flags: []cli.Flag{
```

Comment:
Done! I had to rewrite some of the code tho to make it search recursively up the context, but now it works, and will continue to work in the future no matter how nested the subcommands are. These flags can be placed anywhere in the command.